### ISSUE TITLE
already configured `imports`  in `deno.json` but warning need import_map.json

### ISSUE THREAD

**Issue by JianJroh** [denoland/vscode_deno#833]
**Describe the bug**

**To Reproduce**

 already configured `imports`  in `deno.json`
```json
{
  "tasks": {
    "dev": "deno run --allow-net --allow-read --allow-env --watch app.ts",
    "start": "deno run --allow-net --allow-read --allow-env  app.ts"
  },
  "imports": {
    "@/": "./",
    "dotenv/": "https://deno.land/std@0.182.0/dotenv/"
  },
}
```

 Import error and warning  "Failed to load the import_map.json" .
It seems that an additional import_map.json file is required even if `imports` are configured in `deno.json`.

**Expected behavior**

No need for repeated  imports `imports` configuration

**Versions**

vscode: 1.77.1
deno: 1.32.2 
extension: 3.17.0

**Comment by bartlomieju** [comment]
Hey @JianJroh, sorry for a slow response. Do you experience the same problem with latest Deno (v1.35)?

**Comment by JianJroh** [comment]
> 
The problem cannot be reproduced now, it seems that it has been fixed in the current version (v1.35.1).

**Comment by JianJroh** [comment]
> Hey @JianJroh, sorry for a slow response. Do you experience the same problem with latest Deno (v1.35)?

The problem cannot be reproduced now, it seems that it has been fixed in the current version (v1.35.1).
